src/components-examples/journey-maps/angular/index.ts

```diff
@@ -11,10 +11,28 @@ import { SbbTabsModule } from '@sbb-esta/angular/tabs';
 import { SbbJourneyMapsModule } from '@sbb-esta/journey-maps';
 
 import { JourneyMapsBasicExample } from './journey-maps-basic/journey-maps-basic-example';
-import { JourneyMapsFullExample } from './journey-maps-full/journey-maps-full-example';
+import { JourneyMapsMapNavigationExample } from './journey-maps-map-navigation/journey-maps-map-navigation-example';
+import { JourneyMapsPoisMarkersExample } from './journey-maps-pois-markers/journey-maps-pois-markers-example';
+import { JourneyMapsRoutesZonesExample } from './journey-maps-routes-zones/journey-maps-routes-zones-example';
+import { JourneyMapsStyleOptionsExample } from './journey-maps-style-options/journey-maps-style-options-example';
+import { JourneyMapsUiOptionsExample } from './journey-maps-ui-options/journey-maps-ui-options-example';
 
-export { JourneyMapsBasicExample, JourneyMapsFullExample };
-const EXAMPLES = [JourneyMapsBasicExample, JourneyMapsFullExample];
+export {
```

Comment:
Can we "force" somehow the order of the examples @kyubisation ?
I would like to habe the “SBB UI Options” as second example, after maps-basic.
Then “SBB Map Navigation” and the others.